core/ev-admin.php

```diff
@@ -706,24 +779,21 @@ function delete_author_handler() {
           $post = get_post( $query_video->ID );
           $post->post_status = 'trash';
           wp_update_post($post);
-          $del_videos += 1;
+          $count_trash += 1;
         }
       }
 
       // Save the options without this channel
       if( update_option( 'sp_external_videos_options', $options ) ) {
-        // unset( $_POST['host_id'], $_POST['author_id'] );
 
-        // $message .= printf( __( 'Deleted channel and moved %d video to trash.', 'Deleted channel and moved %d videos to trash.', $del_videos, 'external-videos' ), $del_videos );
+        $message = sprintf( _n( 'Deleted channel %s from %s and moved %d video to trash.', 'Deleted channel %s from %s and moved %d videos to trash.', $count_trash, 'external-videos' ), $_POST['author_id'], $_POST['host_id'], $count_trash );
 
-        $message = "Deleted channel " . $_POST['author_id'] . " from " . $_POST['host_id'] . " and moved " . $del_videos . " videos to trash. ";
-        $message = sp_ev_wrap_admin_notice( $message, 'info' );
-        $messages .= $message;
+        $message = $this->wrap_admin_notice( $message, 'info' );
       }
 
     }
 
-    wp_send_json( $messages );
```

Comment:
Hmm this was there to deal with multiple messages created in this function. Why would we not need that any longer?